commands/games/roulette.js

```diff
@@ -0,0 +1,89 @@
+const { Command } = require('discord.js-commando');
+const stripIndents = require('common-tags').stripIndents;
+
+const Currency = require('../../currency/Currency');
+const Roulette = require('../../Roulette');
+
+const currency = new Currency();
+
+module.exports = class RouletteCommand extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'roulette',
+			aliases: ['roulette', 'bet'],
+			group: 'games',
+			memberName: 'roulette',
+			description: 'Play a game of russian roulette for donuts!',
+			details: 'Play a game of russian roulette for donuts.',
+			throttling: {
+				duration: 30,
+				usages: 1
+			},
+			args: [
+				{
+					key: 'donuts',
+					prompt: 'How many donuts do you want to bet?',
+					type: 'integer'
+				}, {
+					key: 'space',
+					prompt: 'On what space do you want to bet?',
+					type: 'string'
+				}
+			]
+		});
+	}
+
+	async run(msg, args) {
+		const bet = args.donuts;
+		const space = args.space.toLowerCase();
+
+		const balance = await currency.getBalance(msg.author.id);
+		let roulette = Roulette.findGame(msg.guild.id);
+
+		if (balance < 100) {
+			return msg.reply(`you don't have enough 🍩s. You need at least 100 🍩s to bet, but your current account balance is ${balance} 🍩s.`);
```

Comment:
Winnings don't get split. Each user plays individually, with the quirk that multiple users can play at the same time. Said user can bet on only one space in one instance of the game. Upon joining/betting the user loses the donuts they bet. If they win they get their bet * multiplier, if they lose they just lose they get nothing back.